R/frq.R

```diff
@@ -509,6 +509,10 @@ frq_helper <- function(x, sort.frq, weight.by, cn, auto.grp, title = NULL, show.
 
     colnames(mydat) <- c("val", "frq")
 
+    if (!anyNA(suppressWarnings(as.numeric(attr(mydat$val, "levels"))))) {
+      mydat$val <- sjlabelled::as_numeric(mydat$val, keep.labels = F)
+    }
+

```

Comment:
What does this code chunk do?

Reply:
or: why do you think it's needed?

Reply:
When the `val` column is numeric this makes it truly `numeric`, instead of `factor`, as it was already done when there are labels at line 485.

Further, by an aesthetic reason. A `NA` in a character or factor  column in a data frame is printed as `<NA>`, while in a numeric or logic column is printed as `NA`  (always by the `print.data.frame()` method).  Thus, it is more consistent with this.